BrainPortal/config/console_rc/lib/pretty_view.rb

```diff
@@ -342,6 +342,7 @@ def pretview
       bourreau_id.presence || 0, bourreau.try(:name),
       ncpus.presence || 0,
       container_engine.presence || "(none)",
+      container_index_location || "",
```

Comment:
You cannot just add an argument to the sprintf without adjusting the control string accordingly. This will result in out of sync args . Please fix this so that the index is shown as a prefix to the image name on the same line. And please test it before you commit.

Reply:
Right now `pv` doesn't work at all for any ToolConfigs because of the mismatch.

Reply:
Made the change locally - I haven't tested this beyond running the tests (which it passes) because I don't know how to find where this view is exposed in the web app, and cannot find documentation that describes this....

Reply:
I just test it in the console seems to be ok.